docs/proposals/node-removal.md

```diff
@@ -0,0 +1,72 @@
+# Proposal: Node remove
```

Comment:
Please move this from `proposals` to `docs` folder and remove `Proposal` in header.